lib/engine/step/g_1856/buy_sell_par_shares.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require_relative '../buy_sell_par_shares'
+
+module Engine
+  module Step
+    module G1856
+      class BuySellParShares < BuySellParShares
+        def process_par(action)
+          share_price = action.share_price
+          corporation = action.corporation
+          entity = action.entity
+          @game.game_error("#{corporation} cannot be parred") unless corporation.can_par?(entity)
+
+          corporation.par!
```

Comment:
what's the point of overriding this? it looks 99% copy pasted

Reply:
Overriding it lets me call `par!` on the corporation in the middle of the method, I'll refer to this in my response to your comment on corporation.rb

Reply:
The class will be needed for implementing vesting later as well